platform/feature-api/api/src/pod-api/spec.ts

```diff
@@ -108,42 +102,6 @@ export class PodSpec {
                     fc.pre(cont);
                 }
 
-                // Skipped - polyOut.writeFile is not yet used in production
-                it.skip("write/read", async () => {
```

Comment:
Why not keep these tests? From what I understand the API is implemented in the platforms. Why are we skipping the tests? Them not being used in production does not seem like a good enough reason.

Reply:
`write` has not been used in production, as it says here. These tests are skipped, so I might as well delete them.
Actually, the other test below is activated in #989... Another reason we should try and solve this.

Reply:
I don't think it will ever be used. We're using the filesystem, or other ways, to write files; it's safer. But we should probably add a deprecation cycle for the method or something like that, to clarify what's going to happen to it.

Reply:
Created a Jira ticket for this.

Reply:
But this is an interface right? Are features using the filesystem directly? From what I know, that is not the case. If a feature wants to write a file, it needs to go through the pod api. And then the pod uses the filesystem (in the case of pod.js it uses IndexedDB). 

IMO, we should keep this API and tests, even if the API is not used yet.

Reply:
What is the link to the Jira Ticket?